### PR TITLE
Adding unittest 2 Addition + 10

### PR DESCRIPTION
**Pull request by philippethiran**

### CODE REVIEW

File: tests/test_sum/test_another_sum.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import unittest
-from sum.another_sum import another_sum
+from sum.another_sum import another_sum, adD10
```
Comment (philippeunamur) [inline tests/test_sum/test_another_sum.py]: adD10 is not using the correct PEP-8 function naming. Choose add_10 instead of adD10

File: tests/test_sum/test_another_sum.py [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 
 class MyTestCase(unittest.TestCase):
     def test_another_sum(self):
         assert another_sum(3, 2) == 5
 
+    def test_one10(self):
+        assert adD10(3) == 13
```
Comment (philippeunamur) [inline tests/test_sum/test_another_sum.py]: Same problem of function naming. Choose add_10

Comment (philippeunamur) [inline tests/test_sum/test_another_sum.py]: unittest could cover also a test with a negative number.

**Review by philippeunamur — CHANGES_REQUESTED** [review]

File: tests/test_sum/test_another_sum.py (hunk 1 above)
Comment (philippethiran) [reply tests/test_sum/test_another_sum.py]: Good comment: I will change it.

File: tests/test_sum/test_another_sum.py (hunk 2 above)
Comment (philippethiran) [reply tests/test_sum/test_another_sum.py]: Good comment!

Comment (philippethiran) [reply tests/test_sum/test_another_sum.py]: Yep. Will do it

**Review by philippeunamur — APPROVED** [review]
